input/pagecontent/reference.md

```diff
@@ -563,6 +563,8 @@ FHIR allows lists in profiles and extensions to be compartmentalized into sublis
 
 To access a slice of a slice (a resliced array), follow the first pair of brackets with a second pair containing the resliced slice name.
 
+Since slices are sublists, a sliced array path technically points to the *first* item in the sub-list (e.g., index 0 of the slice's sublist). Other items in the sublist can be accessed by appending square-bracketed integer indices (e.g., `[1]`) or soft indices (e.g., `[+]`) to the end of the sliced array path. In this case, indices are relative to the first item in the slice.
```

Comment:
Most places where the slice sublists are referenced, they're written as "sublist" but this sentence contains one "sub-list". I tend to lean toward consistency, so what do you think of this suggestion?

```suggestion
Since slices are sublists, a sliced array path technically points to the *first* item in the sublist (e.g., index 0 of the slice's sublist). Other items in the sublist can be accessed by appending square-bracketed integer indices (e.g., `[1]`) or soft indices (e.g., `[+]`) to the end of the sliced array path. In this case, indices are relative to the first item in the slice.
```